modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1 +1,44 @@
 = What’s New in Studio
+
+Anypoint Studio 7.10 introduces a notification mechanism to help you keep up to date with your modules versions in Exchange, and also enables you to propagate metadata to flows and sub-flows.
```

Comment:
```suggestion
Anypoint Studio 7.10 introduces a notification mechanism to help you keep your module versions in Exchange up to date. Studio 7.10 also enables you to propagate metadata to flows and subflows.
```

Reply:
I separated these because the initial version could imply that the notification mechanism enables you to propagate data.